perun/collect/trace/systemtap_script.py

```diff
@@ -75,44 +75,59 @@ def _end_marker(process):
 #     pass
 
 
+def _begin_marker(process, process_id, samples):
```

Comment:
Suggestion: Rename to "_add_begin_marker". Begin seems to be ambiguous (can serve both as adjective and verb)